### PR TITLE
Girlfriend problem in Week 4, 5, & 6.

### PR DESCRIPTION
**Pull request by MasterDirector99**
I have a pull request for you, Girlfriend from Week 4, 5, & 6 is set to default when you start them, can you try to fix it? Thank you!

### CODE REVIEW

**Comment by asvlzhan** [comment]
muffin will probably close it because it's so lazy to check all the files that you added commits....

**Comment by regulad** [comment]
> 
> 
> muffin will probably close it because it's so lazy to check all the files that you added commits....

Yeah I'm not even sure the author knows what a pull request is.

**Comment by regulad** [comment]
This just seems like they selected a random user & branch

**Comment by Rafraud** [comment]
Hey this is way too many commits for one PR and its really hard to follow. You should close this, organize what you want added or suggested, and submit a more manageable PR than this...

**Comment by Squiddu** [comment]
> If you really want all of this looked at you might be better off splitting your suggested changes into relevant blocks, and submitting it that way, but oof

Guess that won't happen lel

**Comment by Rafraud** [comment]
> Guess that won't happen lel

Guess not. Hope they listen before reviewer comes in

**Comment by Squiddu** [comment]
Please, for the love of God, close this PR
(Also the branch is for FNF Modding Plus if someone doesn't know)

**Comment by VoltrexMaster** [comment]
What the...

**Comment by MasterDirector99** [comment]
Had to close the pull request because i got the wrong one...
